File: Java Fundamentals/Methods/src/PalindromeIntegers.java
import java.util.Scanner;

public class PalindromeIntegers {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        while(true){
            String currentInput = scan.nextLine();
            if(currentInput.equals("END")){
                break;
            }
            int currentNum = Integer.parseInt(currentInput);
            if(isPalindrome(currentNum)){
                System.out.printf("true%n");
            }else{
                System.out.printf("false%n");
            }
        }

    }

    public static boolean isPalindrome(int num){
        String numString = Integer.toString(num);
        StringBuilder reverseNumString = new StringBuilder();

        for (int i = (numString.length()-1); i >= 0; i--) {
            char currentDigit = numString.charAt(i);
            reverseNumString.append(currentDigit);
        }

        return numString.equals(reverseNumString.toString());
    }
}

File: JS Front-End/Functions and Statements/PalindromeIntegers.js
function printPalindromeCheck(nums){

    function isPalindrome(numToCheck){
        let numString = numToCheck.toString();
        let length = numString.length;
        let firstChar;
        let secondChar;
        for(let i = 0; i < length / 2; i++){
            firstChar = numString.charAt(i);
            secondChar = numString.charAt(length - (i + 1));

            if(firstChar != secondChar){
                return false;
            }
        }

        return true;
    }

    for(let number of nums){
        console.log(isPalindrome(number));
    }
}


printPalindromeCheck([123,323,421,121]);
printPalindromeCheck([32,2,232,1010]);